### PR TITLE
Added Attributes to Receipt, Block and BlockHeader Structs

### PR DESCRIPTION
**Pull request by fredfortier**
As discussed in issue #180 , I added optional `mixHash` and `nonce` attributes to the Block and BlockHeader structs. In addition, I added `logsBloom` to the Receipt struct.

### CODE REVIEW

File: src/types/block.rs [hunk 1]
```diff
@@ -98,6 +103,10 @@ pub struct Block<TX> {
     pub transactions: Vec<TX>,
     /// Size in bytes
     pub size: Option<U256>,
+    #[serde(rename = "mixHash")]
```
Comment (tomusdrw) [inline src/types/block.rs]: Missing docs?

Comment (fredfortier) [reply src/types/block.rs]: Sorry, I'll adjust in a minute.

Comment (fredfortier) [reply src/types/block.rs]: Added the missing docstring and changed the type of `nonce` to H64. Here is a reference that supports H64 as the correct type: https://github.com/ethereum/go-ethereum/blob/461291882edce0ac4a28f64c4e8725b7f57cbeae/core/types/block.go:43. Feel free to double check.

Comment (fredfortier) [reply src/types/block.rs]: Fixing the unit test that I broke...

File: src/api/eth.rs [hunk 2]
```diff
@@ -416,6 +418,7 @@ mod tests {
     "cumulativeGasUsed": "0x33bc",
     "gasUsed": "0x4dc",
     "contractAddress": "0xb60e8dd61c5d32be8058bb8eb970870f07233155",
```
Comment (fredfortier) [inline src/api/eth.rs]: Your `EXAMPLE_RECEIPT` states that it was taken from the Ethereum JSON-RPC docs but did not include `logsBloom`. I verified the docs: https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_gettransactionreceipt ; I do see a value for `logsBloom`, so I included it. I'm not sure if there was a specific reason for excluding it. All tests are now passing on my side.

**Pull request merged**